scripts/templates/EmptyTypes.mustache

```diff
@@ -0,0 +1,45 @@
+import { IStyle } from 'office-ui-fabric-react/lib/Styling';
+import { IComponent, IStyleableComponentProps } from '../../Foundation';
+
+export type I{{componentName}}Component = IComponent< I{{componentName}}Props, I{{componentName}}ViewProps, I{{componentName}}Styles>;
+
+// Extending IStyleableComponentProps will automatically add stylable props for you, such as styles and theme.
+//    If you don't want these props to be included in your component, just remove this extension.
+export interface I{{componentName}}Props extends IStyleableComponentProps< I{{componentName}}ViewProps, I{{componentName}}Styles> {
+  // All props for your component are to be defined here.
+  /**
+   * Component sample default prop for use if component is uncontrolled.
+   * @default 'Default Text'
```

Comment:
Due to upcoming API extractor integration, this will need to be changed to @defaultvalue.